Source/monster.cpp

```diff
@@ -3659,8 +3653,7 @@ void M_StartHit(Monster &monster, const Player &player, int dam)
 
 void MonsterDeath(Monster &monster, Direction md, bool sendmsg)
 {
-	if (!monster.isPlayerMinion())

```

Comment:
why did you remove this?

Reply:
It was removed by accident during merging, it is now reverted to how it should be